javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,39 @@
+     // All at once
+     function translateAllAtOnce(){
+        
+        const promise1 = moveElement(document.querySelector('li:nth-child(1)'), {x: 20, y: 300}) 
+    
+        const promise2 =  moveElement(document.querySelector('li:nth-child(2)'), {x: 400, y: 300})
+        
+        const promise3 =   moveElement(document.querySelector('li:nth-child(3)'), {x: 400, y: 20})
+        
+        return(promise1,promise2,promise3)
+        
+        .then(() => {
+            console.log('Element has been moved');
+        })
+     }
```

Comment:
Could you do this using `Promise.all()` ([link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all))?